operator/roles/kiali-deploy/templates/openshift/console-links.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleLink
+metadata:
+  name: kiali-app-link
+spec:
+  href: {{ kiali_route_url }}
+  location: ApplicationMenu
+  text: Kiali
+{% for namespace in kiali_vars.deployment.accessible_namespaces %}
+---
+apiVersion: console.openshift.io/v1
+kind: ConsoleLink
+metadata:
+  name: "kiali-namespace-{{ namespace }}-link"
```

Same as above - need more metadata:

```suggestion
  name: "kiali-namespace-{{ namespace }}-link"
  namespace: {{ kiali_vars.deployment.namespace }}
  labels:
    app: kiali
    version: {{ kiali_vars.deployment.version_label }}
```

---

I'm not exactly sure if the `ConsoleLink` has a namespace, but will merge those and see what happens.